src/components/MenuItems/MenuItems.tsx

```diff
@@ -10,13 +11,25 @@ export const MenuItems = ({ products }: Props): ReactElement => {
   if (products.length === 0) {
     return <div />;
```

Comment:
does this need to return a div or would `null` be fine?

Reply:
Yeah it needs to return a react element